src/components/preference-item/preference-item.component.html

```diff
@@ -5,5 +5,10 @@
   <ng-container [ngSwitch]="type">
     <input *ngSwitchCase="'number'" type="number" [(ngModel)]="value"/>
     <mat-slide-toggle *ngSwitchCase="'boolean'" [(ngModel)]="value"></mat-slide-toggle>
+    <mat-select *ngSwitchCase="'select'" [(ngModel)]="value">
+      <mat-option *ngFor="let option of options" [value]="option">
+        {{option}}
+      </mat-option>
+    </mat-select>
```

Comment:
Just for future PRs - Adding the mat-select and modifying the PreferenceGroup / PreferenceItem would be a nice distinct PR for review. Totally get that sometimes features come up in the middle of other tasks, but just wanted to note this for the future (mostly so we can hopefully churn through reviews a bit faster :D ). Thanks!

Reply:
No problem, sorry for the inconvenience. I will break them into smaller PRs in my future work.